readme.md

```diff
@@ -41,7 +41,7 @@ var theBody = document.querySelectorAll("body");
 theBody.style.backgroundColor = "red";
 ```
 
-> Your answer...
+> The query selected was named wrong. It should be "theBody".
 
```

Comment:
This won't work because `querySelectorAll` returns an array of DOM objects. This is true even if there is only one of a given element in the DOM (like `body`). You cannot call `.style` on an array of DOM objects - only on a particular DOM object.

Alternate ways to do this are by setting `theBody` to...
```js
document.body
document.querySelectorAll("body")[0]
document.querySelector("body")
```